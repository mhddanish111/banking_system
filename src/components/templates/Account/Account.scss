@import "../../../styles/colors.scss";
@import "../../../styles/fonts.scss";
@import "../../../styles/mixins.scss";
@import "../../../styles/spacing.scss";
.account-container {
    display: flex;
    .transaction-handler {
      display: flex;
      justify-content: space-between;
    }
    .account-details-container {
        display: flex;
        background-color: $background-color_2;
        padding: 20px;
        @include mobile{
          flex-direction: column;
        }
        .account-details-box {
          @include desktop {
            margin: $space-s;
          }
          @include tablet-and-under {
              margin-top: $space-s;
          }
          word-wrap: break-word;
        }
        label {
          font-weight: $font-weight-bold;
        }
      }
}