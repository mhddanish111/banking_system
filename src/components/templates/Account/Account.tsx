import React from 'react';
import Layout from '../Layout';
import AccountTransaction from '../../organisms/AccountTransaction';
import './Account.scss';

const Account: React.FC<any> = () => {
  return (
    <Layout pageHeading="Account">
      <AccountTransaction transactionType="Accounts" />
    </Layout>
  );
};

export default Account;
