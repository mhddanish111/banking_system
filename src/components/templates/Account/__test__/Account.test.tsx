import { shallow } from 'enzyme';
import Account from '../Account';

describe('<Account />', () => {
  const wrapped = shallow(<Account />);
  it('should render the Account Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });
});
