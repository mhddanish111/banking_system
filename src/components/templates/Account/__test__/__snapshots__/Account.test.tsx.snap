// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Account /> should render the Account Component correctly 1`] = `
<Layout
  pageHeading="Account"
>
  <AccountTransaction
    transactionType="Accounts"
  />
</Layout>
`;
